canvas_modules/common-canvas/src/common-properties/common-properties.jsx

```diff
@@ -266,7 +266,8 @@ CommonProperties.propTypes = {
 		validationHandler: PropTypes.func,
 		titleChangeHandler: PropTypes.func,
 		propertiesActionLabelHandler: PropTypes.func,
-		tooltipLinkHandler: PropTypes.func
+		tooltipLinkHandler: PropTypes.func,
+		iconCallBack: PropTypes.func
```

Comment:
Maybe rename this to propertyIconHandler.  We already have a buttonIconHandler so this kind of matches that.